scripts/prepare_alpaca.py

```diff
@@ -54,6 +54,7 @@ def prepare(
     tokenizer = Tokenizer(checkpoint_dir)
 
     # Partition the dataset into train and test
+    test_split_size = len(data) * test_split_percent // 100
```

Comment:
`random_split` supports having lengths as percentages: https://pytorch.org/docs/stable/data.html#torch.utils.data.random_split
which means that we can simplify this further

Reply:
Cool!  I didn't know that.